src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const amountPerDay = 40;
+  const totalAmount = days * amountPerDay;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days >= 3 && days < 7) {
+    return totalAmount - smallDiscount;
+  }
+
+  if (days >= 7) {
```

Comment:
Avoid using magic numbers - it's a programming antipattern. Use variables with meaningful names instead